cartography/intel/aws/ec2/instances.py

```diff
@@ -71,8 +71,8 @@ def load_ec2_instance_network_interfaces(neo4j_session: neo4j.Session, instance_
 
 @timeit
 def load_ec2_instances(
-    neo4j_session: neo4j.Session, data: List[Dict], region: str, current_aws_account_id: str,
-    update_tag: int,
+        neo4j_session: neo4j.Session, data: List[Dict], region: str, current_aws_account_id: str,
```

Comment:
plz don't change spacing

Reply:
This was again as a result of the linter. For some reason the linter failed due to the previous indentation formatting. Passed for this one.